src/library_manager/lib_manager.c

```diff
@@ -155,26 +155,13 @@ static void __sparse_cache *lib_manager_get_instance_bss_address(uint32_t module
 								 uint32_t instance_id,
 								 struct sof_man_module *mod)
 {
-	uint32_t instance_bss_size =
-		 mod->segment[SOF_MAN_SEGMENT_BSS].flags.r.length / mod->instance_max_count;
-	uint32_t inst_offset = instance_bss_size * PAGE_SZ * instance_id;
-	void __sparse_cache *va_base =
-		(void __sparse_cache *)(mod->segment[SOF_MAN_SEGMENT_BSS].v_base_addr +
-					inst_offset);
-
-	tr_dbg(&lib_manager_tr,
-	       "lib_manager_get_instance_bss_address(): instance_bss_size: %#x, pointer: %p",
-	       instance_bss_size, (__sparse_force void *)va_base);
-
-	return va_base;
+	return (void __sparse_cache *)mod->segment[SOF_MAN_SEGMENT_BSS].v_base_addr;
```

Comment:
This change will break IADK originated modules compatibility. IADK modules will not work in multiple instances.

Reply:
@lyakh this now resolved  ?

Reply:
@jxstelter fixed by keeping the system-service version unchanged